frontend/src/components/formsContainer/FormsContainer.cy.js

```diff
@@ -0,0 +1,14 @@
+// import FormsContainer from "./FormsContainer";
+
+// describe('FormsContainer', () => {
+//   beforeEach(() => {
+//     cy.visit('/'); 
+//   });
+
+//   it('should select a character', () => {
+//     const selectedCharacter = 'Mickey Mouse';
+
+//     cy.get('#character').select(selectedCharacter);
+//     cy.get('p').should('have.text', `Selected: ${selectedCharacter}`);
+//   });
+// });
```

Comment:
Is this a unit test or a full e2e? If this is a unit test (I am assuming it is), we can just mount a forms container component and cross check whether the different options we specified exist as options in the dropdown menus (we dont need to visit ('/') as we can mount this component independent to a webpage.)